src/rfac/rfac.c

```
@@ -0,0 +1,277 @@
+  args->vi = 4.0;                   /* im. part of opt. potential */
+
+  strcpy(args->iv_file, "\0");             /* no output of IV curves */
+  strcpy(args->out_file, "\0");            /* will be treated as "stdout" */
+
+  return (args);
+}
+
+/*!
+ * Free #rfac_iv structure instance from memory.
+ */
+void rfac_iv_free(rfac_iv *_iv)
```

Cppcheck (2.19.0)

[style] The function 'rfac_iv_free' should have static linkage since it is not used outside of its translation unit.

(staticFunction)